static/app/views/dashboardsV2/widgetBuilder/sortBySelectors.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled';
+
+import SelectControl from 'sentry/components/forms/selectControl';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {SelectValue} from 'sentry/types';
+
+import {SortAscOrDescOption, sortAscOrDescOptions} from './utils';
+
+interface Values {
+  sortAscOrDesc: SortAscOrDescOption;
```

Comment:
I don't have a strong opinion on this but I was wondering if we could call this `sortDirection` instead. My reasoning is that it feels a little too concrete to be referring to asc/desc especially if the enum values are actually "high_to_low"/"low_to_high" and using that wording